specification/service-map/resource-manager/Microsoft.OperationalInsights/preview/2015-11-01-preview/arm-service-map.json

```diff
@@ -2605,14 +3063,14 @@
             "$ref": "#/definitions/Process"
           }
         },
-        "Ports": {
+        "ports": {
           "description": "Port resources.",
           "type": "array",
           "items": {
             "$ref": "#/definitions/Port"
           }
         },
-        "ClientGroups": {
+        "clientGroups": {
```

Comment:
Could you confirm this is expected and the **exact** case used in the JSON?

Reply:
yes, they are.